splinepy/io/json.py

```diff
@@ -0,0 +1,153 @@
+"""
+Export splines in custom json format
+"""
+
+import base64
+import json
+import logging
+import numpy as np
+
+def load(fname):
+    """
+    Loads splines from json file. Returns dict-splines
+
+    Parameters
+    -----------
+    fname: str
+
+    Returns
+    --------
+    spline_list: list
+    """
+    # Import data from file into dict format
+    jsonbz = json.load(open(fname, "r"))
+
+    spline_list = []
+    base64encoding = jsonbz["Base64Encoding"]
+
+    # Init return type
+    return_dict = {
+        "Bezier" : [],
+        "NURBS" : [],
+        "BSpline" : []
+    }
+
+    for jbz in jsonbz["SplineList"]:
+
+        # Convert Base64 str into np array
+        if base64encoding:
+            jbz["control_points"] = \
+                np.frombuffer(
+                  base64.b64decode(
+                    jbz["control_points"].encode('ascii')
+                  ), dtype=np.float
+                ).reshape((-1,jbz["dim"]))
+
+        # Transform data into Splines
+        if jbz["SplineType"] == "Bezier":
+            return_dict["Bezier"].append(
+                {
+                    "control_points" : jbz["control_points"],
+                    "degrees" : jbz["degrees"]
+                }
+            )        
+        elif jbz["SplineType"] == "NURBS":
+            return_dict["NURBS"].append(
+                {
+                    "control_points" : jbz["control_points"],
+                    "degrees" : jbz["degrees"],
+                    "weights" : jbz["weights"],
+                    "knot_vectors" : jbz["knot_vectors"]
+                }
+            )        
+        elif jbz["SplineType"] == "BSpline":
+            return_dict["BSpline"].append(
+                {
+                    "control_points" : jbz["control_points"],
+                    "degrees" : jbz["degrees"],
+                    "knot_vectors" : jbz["knot_vectors"]
+                }
+            )        
+        else :
+            logging.warning("Found unknown spline-type: " + str(jbz))
+    logging.debug("Imported " + str(len(spline_list)) + " splines from file.")
+    return return_dict
+
+def export_splines(fname, spline_list, list_name=None, base64encoding=False):
```

Comment:
as mentioned above, my TODO/goal was to have `load` and `export` only. Can you combine `export_splines` and `write_json`?